source/draw/gpu/opengl/OpenGLSurface.ooc

```diff
@@ -37,7 +37,7 @@ OpenGLSurface: abstract class extends GpuCanvas {
 		map model = this _createModelTransform(destination)
 		map view = this _view
 		map projection = this _projection
-		map use()
+		map use(null)
```

Comment:
It might be possible to call a function that is overridden in `OpenGLCanvas` but adding complexity to support a safety feature in legacy code is not worth it.